week_six/quiz_class.dart

```diff
@@ -14,7 +14,17 @@ void main() {
 }
 
 List<int> findPair(List<int> numbers, int target) {
-  // Your code here
-
-  return [2, 5];
+  List<int> found=[];
+  for(int count=0;count<numbers.length;count++){
+    for(int count1=count ; count1<numbers.length;count1++){
+      if(numbers[count] + numbers[count1]==target){
```

Comment:
Looks good. 
Please name the variable more well.

Reply:
ok sir